### DISCUSSION TITLE
How to draw reactions in black/white or gray scale?

### DISCUSSION THREAD

**Discussion by ddsneo4j** [rdkit/rdkit#5885] [General] [answered]
Hi,

  I have only found an example on how to draw structures with RdKit in black white, but have not been able to find
any method to draw **reactions** (rxn object) in black/white or gray scale.

Any help on this would be highly appreciated.

Thanks,
Dan

**Comment by vfscalfani** [comment, marked as answer]
Hi @ddsneo4j , try something like this and see if it is what you were looking for. The key piece is to set the `drawOptions().useBWAtomPalette()` option:


```
rxn_smi = "CCCCO.Br>>CCCCBr.O"
drawer = rdMolDraw2D.MolDraw2DCairo(900,300)
drawer.drawOptions().useBWAtomPalette()
drawer.DrawReaction(rdChemReactions.ReactionFromSmarts(rxn_smi, useSmiles=True))
drawer.FinishDrawing()
bio = io.BytesIO(drawer.GetDrawingText())
Image.open(bio)
```

Hope it helps. 

Vin

**Reply by ddsneo4j** [reply]
Thanks, that works!
